audb/core/api.py

```diff
@@ -602,8 +600,11 @@ def versions(
                                 header = p.joinpath(f"db-{version}.yaml")
                                 if header.exists():
                                     vs.extend([version])
-                    except artifactory.ArtifactoryException:  # pragma: nocover
-                        # This tackles the case of missing read permissions.
+                    except Exception:  # pragma: nocover
```

Comment:
Ok. There is probably no reason to distinguish between ` artifactory.ArtifactoryException` that afaics still could be raised and the the catch all Èxception`.